lib/testers/rule-tester.js

```diff
@@ -490,7 +490,19 @@ RuleTester.prototype = {
                 }
 
                 if (item.hasOwnProperty("output")) {
-                    const fixResult = SourceCodeFixer.applyFixes(eslint.getSourceCode(), messages);
+                    let fixResult = SourceCodeFixer.applyFixes(eslint.getSourceCode(), messages);
+
+                    if (item.multipass) {
+                        for (let i = 0; i < 10; i++) {
```

Comment:
Makes sense. Really, seems like it would be cleanest if this code could actually call `multipassFix`, but seemed like that would probably be hard (although looking at it now, maybe not).

If we decide to keep this part, I can use a shared constant, I guess on `CLIEngine`.